01_basic_stream/README.md

```diff
@@ -0,0 +1,135 @@
+# Simple stream
+
+This tutorial will get you started with the basic of using Anbox Cloud. By the end you will
+have a custom web page capable of streaming Android 10.
+
+## 1. Creating the application
```

Comment:
We miss instructions from where to access AMS. Think about somebody reading this who has just deployed the appliance or the regular Anbox Cloud. He has no clue what amc or ams is and just wants to copy&paste commands step by step

Reply:
Ah right, there was a `00_prerequisites` before that had those steps but that was removed